### ISSUE TITLE
Compile error in Elixir for 0.1.0

### ISSUE THREAD

**Issue by stephanos** [opentelemetry-beam/opentelemetry_zipkin#4]
When I add

```
{:opentelemetry_zipkin, "~> 0.1.0", only: :test}
```

to my mix file and run `mix test`, I get:

```
===> Compiling src/opentelemetry_zipkin.erl failed
src/opentelemetry_zipkin.erl:81: field time undefined in record event
src/opentelemetry_zipkin.erl:84: variable 'Timestamp' is unbound

** (Mix) Could not compile dependency :opentelemetry_zipkin, "/Users/stephan/.asdf/installs/elixir/1.10.2/.mix/rebar3 bare compile --paths="/Users/stephan/my-service/_build/test/lib/*/ebin"" command failed. You can recompile this dependency with "mix deps.compile opentelemetry_zipkin", update it with "mix deps.update opentelemetry_zipkin" or clean it with "mix deps.clean opentelemetry_zipkin"
```

I'm using Elixir 1.10.2 and Erlang 22.2.8.

**Comment by tsloughter** [comment]
Thanks for the report. Some changes were just released that were backwards incompatible. I've been trying to get the SDK change merged and published to hex but the circleci->github status updates have been broken, s only the API has been published :(.

If you pin to opentelemetry 0.3.0 and opentelemetry_api 0.3.0 in your mix.exs it should work.

**Comment by stephanos** [comment]
thank you, I'll try that!

**Comment by stephanos** [comment]
Cool, that worked. Feel free to close the issue or leave it open.

**Comment by tsloughter** [comment]
I just published 0.2.0 so closing now.
